robot.py

```diff
@@ -230,13 +230,11 @@ def teleopPeriodic(self):
         release_pistons = self.driver.getBackButton() 
 
         if activate_pistons:
-            self.lift.lower_center()
-            self.lift.lower_back()
+            self.lift.raise_all
             self.logger.info("Raising all!")
 
         elif release_pistons:
-            self.lift.raise_back()
-            self.lift.raise_center()
+            self.lift.lower_all
```

Comment:
You'll need to take a look at the implementation of raise_all and lower_all to make sure that they do what you need.   (I think you'll be surprised when you find out why this code is the way that it is...)